js/main.js

```diff
@@ -1,52 +1,94 @@
-// !!!  ДЗ 17. Функція із замиканням
+// !!  ДЗ 20. Створюємо сутності
 
-// ??? Написати функцію, яка приймає 1 параметр. з тим, що передали перший раз і т. д. Все це із замиканнями, наприклад: sum(3) = 3 sum(5) = 8 sum(20) = 28
+// Створити сутність людини:
+// імʼя
+// вік
+// Метод виведення даних
+// Створити сутність автомобіля:
+// Характеристики автомобіля окремими властивостями
+// Методи:
+// Виведення на екран даних про цей автомобіль
+// Привласнення цього автомобіля власнику (записати в автомобіль обʼєкт власника)
 
-// !!!  Рішення
+// !! Рішення
 
-// !!!  варіант 1
+// Що потрібно зробити
 
-// const sum = (function () {
-//     let result = 0; 
-//     return function(number) {
-//        result += number; 
-//        return result; 
-//     } })();
+// 1. Створити клас Людина
+//     властивості 
+//         імʼя
+//         вік
+//     методи
+//         конструктор, який приймає два параметри: імʼя та вік
+//         метод, який виводить у консоль інформацію про людину
 
-// console.log(sum(3));
-// console.log(sum(5));
-// console.log(sum(20));
+class Person {
+  name;
+  age;
 
-// чи так, якщо привести функцію до const і далі вже її виводити
-// як краще???
+  constructor (name, age) {
+    this.name = name;
+    this.age = age;
+  }
+  getInfo() {
```

Comment:
```suggestion
  logInfo() {
```
метод нічого не повертає